controllers/backup.go

```diff
@@ -0,0 +1,274 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	ranv1alpha1 "github.com/openshift-kni/cluster-group-upgrades-operator/api/v1alpha1"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Backup states
+const (
+	BackupStatePreparingToStart = "PreparingToStart"
+	BackupStateStarting         = "Starting"
+	BackupStateActive           = "Active"
+	BackupStateSucceeded        = "Succeeded"
+	BackupStateTimeout          = "BackupTimeout"
+	BackupStateError            = "UnrecoverableError"
```

Comment:
nit: If you're updating this file, sorting these alphabetically would be nice

Reply:
Kept it that way to show the flow of the status PreparingToStart -> Starting -> Active -> Final Status

Reply:
Ok